handlers/challenge_handler.py

```diff
@@ -740,6 +740,8 @@ def update_func(ctf):
 
         if ctf:
             ChallengeHandler.update_ctf_purpose(slack_wrapper, ctf)
+            if ctf.cred_url:
+                ChallengeHandler.set_ctf_topic(slack_wrapper, ctf)
```

Comment:
Pass in `ctf.cred_url` instead of `ctf`. This way, the function could be used to set arbitrary topics, e.g. `set_ctf_topic(slack_wrapper, "yo!")`